pdns-dynamic-reverse-backend.py

```diff
@@ -48,10 +54,10 @@
 import yaml
 
 LOGLEVEL = 2
-CONFIG = 'dynrev.yml'
+CONFIG = '/etc/pdns/dynrev.yml'
```

Comment:
You passed this file path as argument in config `pipe-handler` option. Use that (`argv[1]`) instead IMO.